tests/qam-kgraft/update_kgraft.pm

```diff
@@ -0,0 +1,164 @@
+# SUSE's openQA tests
+#
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+use base 'opensusebasetest';
+use testapi;
+use qam;
+use utils;
+
+use strict;
+use warnings;
+
+sub timemark {
+    my ($sec, $min, $hour, $mday, $mon, $year, $wday, $yday, $isdst) = localtime(time);
+    return sprintf("%02d%02d%02d.%02d%02d.%02d", $year % 100, $mon + 1, $mday, $hour, $min, $sec);
```

Comment:
Why another dependency on thirdparty module for simple stamp ? IT can be anything ...UUID, 

consumer of this is TESTER , not computer